docs/language/overview.md

```diff
@@ -288,8 +287,7 @@ or
 summarize salary:=sum(income) by address:=lower(address)
 ```
 This style of "assignment" to a record value is distinguished from the `=`
-token which binds a locally scoped name to a value that can be referenced
-in later expressions.
+token which is used in [constant](#3-const-statements) and [variable](#8-lateral-subqueries) assignments.
```

Comment:
I wouldn't make this change because the language I had refers generically to all these cases and as Noah points out, we were going to take it out for `type` (but I wasn't sure we should take it out for `const`).  Currently, you can say this for example...
```
over foo with x => (
  const y = 10
  yield this+x+y
)
```

Reply:
@mccanne: Ok, I interpret your comment as declining @nwt's proposal, but I'd therefore appreciate your take on my original proposed change. I agree that your original generic wording seemed accurate, but my motive for linking to the "constant" & "variable" sections is because I sometimes get nervous that certain words might only be clear to people with language chops ("token", "locally scoped"). Given it's a language overview doc, I suppose that's fair game, but I find linking to examples can sometimes help less-experienced audiences connect the dots. But if you feel it becomes inaccurate once changed from you original text, I'd be fine with restoring that.

Reply:
@nwt: Ok, I've pushed a commit putting back the original text then. I still think it could use some improvement, but it doesn't have to happen now. Since you guys made it sound like `=` may be phased out in other contexts soon anyway, happy to wait for the dust to settle.